bin/xbundle_convert

```diff
@@ -1,69 +1,64 @@
 #!/usr/bin/env python
 
-"""
-A command-line script to use xbundle.
+"""xbundle_convert
+
+Description:
+    Convert between OLX and xbundle XML formats.
+    If the input format in an XML file, the output will be OLX.
+    If the input format is a directory, the output will be xbundle.
+
+Usage:
+    xbundle_convert convert [--force-studio] <input> <output>
+    xbundle_convert test
+    xbundle_convert --help
+    xbundle_convert -h
```

Comment:
I think `xbundle_convert -h | --help` should work instead of having two separate command paths.